### PR TITLE
Refactor templates conversation view

### PR DESCRIPTION
**Pull request by owencjones**
# Motivation and Context
Split out templates for secure conversation views

# What has changed
* `conversation-view.html` file moved into dedicated directory
* Elements split into prudent separate parts for verbosity
* View file for conversation view changed to refer to new base file

# How to test?
Mostly doesn't need testing - some acceptance tests may fail, but will be addressed in the main branch rather than here - ensure code quality is good, unit tests pass, and that solution starts, conversations can be run

### CODE REVIEW

File: response_operations_ui/templates/conversation-view/cv-handle-errors.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{% if form.errors %}
+    {% set error_count = form.errors | length %}
+    {% set plural_suffix = 's' if error_count > 1 else '' %}
+    {% set header_text = 'This page has' ~ error_count ~ 'error' ~ plural_suffix %}
```
Comment (NickGrantONS) [inline response_operations_ui/templates/conversation-view/cv-handle-errors.html]: spaces after `has` and before `error` needed or else it'll read "This page has20errors"?

File: response_operations_ui/templates/conversation-view/cv-reply-form.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<div class="secure-message-form">
+    <form action="{{ url_for('messages_bp.view_conversation', thread_id=messages[0].thread_id, page=page) }}" method="post" id="create-message-form">
+        {{ form.csrf_token }}
+        {{ form.subject(id='secure-message-subject', type='hidden', value=subject) }}
+        {{ form.hidden_to_ru_id(type='hidden', value=ru_id) }}
+        {{ form.hidden_survey_id(type='hidden', value=survey) }}
+        {{ form.hidden_to_uuid(type='hidden', value=to_uuid)}}
```
Comment (NickGrantONS) [inline response_operations_ui/templates/conversation-view/cv-reply-form.html]: kinda unrelated, but I hope to god we're not using anything like this in frontstage secure messaging

Comment (owencjones) [reply response_operations_ui/templates/conversation-view/cv-reply-form.html]: I don't know, to be honest.  Do you mean for security reasons?

Comment (NickGrantONS) [reply response_operations_ui/templates/conversation-view/cv-reply-form.html]: yes

**Pull request merged**